src/components/ChatPage/ChatPage.jsx

```diff
@@ -75,6 +83,33 @@ class ChatPage extends React.Component {
 
     }
 
+    handleColorChange(color){
+        var messages = this.state.messages;
+
+        let isHex = /(^#[0-9A-F]{6}$)|(^#[0-9A-F]{3}$)/i.test(color);
```

Comment:
Should we be switching all variable definitions from `var` to `let`? I lowkey still don't understand the difference. I assume it's just a style thing to stay a bit more modern but not too sure.